sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -98,6 +100,10 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
   private val inputFilePath = new File(baseResourcePath, "inputs").getAbsolutePath
   private val goldenFilePath = new File(baseResourcePath, "results").getAbsolutePath
 
+  override def sparkConf: SparkConf = super.sparkConf
```

Comment:
This should be protected, right? In the parent trait SharedSparkSession, it is protected. 

```
protected override def sparkConf ....
```

Reply:
Done in #26019.

Reply:
Yes, seems adding a `protected` modifier is correct, though, we don't usually make separate PRs to address minor nits. Since the PR is open, I guess it's fine.

Reply:
Yeah, copy that.